air/src/options.rs

```diff
@@ -246,6 +249,11 @@ impl ProofOptions {
     pub fn partition_options(&self) -> PartitionOptions {
         self.partition_options
     }
+
+    /// Returns the `[BatchingType]` used in this instance of proof options for the DEEP polynomial.
+    pub fn batching_type_deep(&self) -> BatchingType {
+        self.batching_deep
+    }
```

Comment:
nit: I would probably name this `batching_deep()` or `deep_batching()`.

Also, would the following description be a bit more precise:
```rust
/// Returns the `[BatchingType]` which defines how linear combinations are to be computed during
/// the composition of the DEEP polynomial.
///
/// Linear batching implies that independently drawn random values will be used for random linear
/// combination computation, while Algebraic batching implies that powers of a single random
/// value will be used.
///
/// Depending on other parameters, Algebraic batching may lead to a small reduction in the security
/// level of the generated proofs, but avoids extra calls to the random oracle (i.e., hash function).
```

Reply:
Edited a bit and added

Reply:
Used `batching_used_for_deep_poly` instead, but can change as it is not a strong preference